src/main/java/org/opentripplanner/model/plan/RideOffer.java

```diff
@@ -0,0 +1,84 @@
+package org.opentripplanner.model.plan;
+
+import org.opentripplanner.model.Operator;
+import org.opentripplanner.model.base.ToStringBuilder;
+import org.opentripplanner.routing.core.Money;
+
+/**
+ * This is an offer made by a 3rd party operator it can be attached to an itinerary Leg.
+ * The service can be provided by a Transit, Taxi or TNC Operator.
+ * <p>
```

Comment:
This is dynamic offer
provided by a Minibus or Taxi Service, including services like Uber and Lyft.